### DISCUSSION TITLE
Custom Collection String Representation

### DISCUSSION THREAD

**Discussion by theHamdiz** [pocketbase/pocketbase#4821] [Polls]
I wanted to know if it's possible to register a custom string representation for collections that if not found should display the current behavior shown below, otherwise it shows the value set earlier.

![image](https://github.com/pocketbase/pocketbase/assets/6674568/dd1b6600-9224-41f2-8a88-b12e9ad5d375)

I think it will make navigating the admin UI easier.

**Comment by ganigeorgiev** [comment]
I don't understand what is the purpose of the poll.

If you have suggestion for better "empty" representation instead of "N/A" feel free to elaborate, but there are no plans to add option in the Admin UI for this.

Also please note that the current "Presentable" field option will be most likely removed after the Admin UI refactoring as part of https://github.com/pocketbase/pocketbase/discussions/4355 as there are plans to add shortcut/show the entire record on rel item click/hover.
